### PR TITLE
[IMP] iot_template_oca: add iot.device.configure access rules

### PR DESCRIPTION
**Pull request by olgamarcocb**
On the migration, I forgot to add the security of the wizard.

Fixing it here. Sorry for the trouble.

### CODE REVIEW

**Review by lubusax — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Why do you need this?

**Comment by etobella** [comment]
Wizards must have security groups and this one was missing

File: iot_template_oca/security/ir.model.access.csv [hunk 1]
```diff
@@ -7,3 +7,4 @@ access_iot_template_output,access_iot_template_output,model_iot_template_output,
 manage_iot_template_output,access_iot_template_output,model_iot_template_output,iot_oca.group_iot_manager,1,1,1,1
 access_iot_template_key,access_iot_template_key,model_iot_template_key,iot_oca.group_iot_user,1,0,0,0
 manage_iot_template_key,access_iot_template_key,model_iot_template_key,iot_oca.group_iot_manager,1,1,1,1
+access_iot_device_configure,access_iot_device_configure,model_iot_device_configure,base.group_user,1,0,0,0
```
Comment (etobella) [inline iot_template_oca/security/ir.model.access.csv]: Iot manager better

**Review by etobella — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
A healthy habit is to give these explanations about the whys in the commit message extended text.

File: iot_template_oca/security/ir.model.access.csv [hunk 2]
```diff
@@ -7,3 +7,4 @@ access_iot_template_output,access_iot_template_output,model_iot_template_output,
 manage_iot_template_output,access_iot_template_output,model_iot_template_output,iot_oca.group_iot_manager,1,1,1,1
 access_iot_template_key,access_iot_template_key,model_iot_template_key,iot_oca.group_iot_user,1,0,0,0
 manage_iot_template_key,access_iot_template_key,model_iot_template_key,iot_oca.group_iot_manager,1,1,1,1
+access_iot_device_configure,access_iot_device_configure,model_iot_device_configure,iot_oca.group_iot_manager,1,0,0,0
```
Comment (etobella) [inline iot_template_oca/security/ir.model.access.csv]: permissions should be 1,1,1,1

**Review by etobella — CHANGES_REQUESTED** [review]

File: iot_template_oca/security/ir.model.access.csv (hunk 2 above)
Comment (olgamarcocb) [reply iot_template_oca/security/ir.model.access.csv]: Done!

**Review by etobella — APPROVED** [review]

**Comment by olgamarcocb** [comment]
@pedrobaeza sorry for the trouble. I will improve my commit messages.

**Pull request merged**